packages/ceramic-core/src/pin/level/level-pinning-service.ts

```diff
@@ -0,0 +1,134 @@
+import levelup from "levelup";
+import leveldown from "leveldown";
```

Comment:
Does leveldown work in browsers? We need to be careful with the dbs that we use so that we get full compatibility.